.env.appStore.example

```diff
@@ -87,12 +87,17 @@ VITAL_REGION="us"
 # Used for the Zapier integration
 # @see https://github.com/calcom/cal.com/blob/main/packages/app-store/zapier/README.md
 ZAPIER_INVITE_LINK=""
-# *********************************************************************************************************
 
 # - LARK
 # Needed to enable Lark Calendar integration and Login with Lark
 # @see <https://open.larksuite.com/document/ukTMukTMukTM/ukDNz4SO0MjL5QzM/g>
 LARK_OPEN_APP_ID=""
 LARK_OPEN_APP_SECRET=""
 LARK_OPEN_VERIFICATION_TOKEN=""
+
+#   - WEB3
+# Used for the Web3 plugin
+# @see https://github.com/calcom/cal.com/blob/main/packages/app-store/web3/README.md
+ALCHEMY_API_KEY=""
+INFURA_API_KEY=""
```

Comment:
@calcom/reviewers added INFURA_API_KEY to vercel environments. login to infura is in password manager